packages/components/src/button/style.scss

```diff
@@ -75,7 +75,7 @@
 		&:disabled:active:enabled,
 		&[aria-disabled="true"],
 		&[aria-disabled="true"]:enabled, // This catches a situation where a Button is aria-disabled, but not disabled.
-		&[aria-disabled="true"]:active:enabled {
+		&[aria-disabled="true"]:active:enabled:not(.is-busy) {
```

Comment:
This works as intended and no longer changes the hover state of busy buttons.

However, I'm not sure I agree with it. I think busy buttons should still have a hover state, and that hover state should just respect the busy button styles. 

Although most component libraries make loading/busy buttons disabled, I can definitely see a valid case for a busy button that's still not disabled. Maybe an optimistic "Add to Cart" button that remains enabled? Or any other counter-like button that counts something inline but also submits data to the server? We also have `__experimentalIsFocusable` so that fixes the a11y concerns about disabled busy buttons.

In simple terms, I'd expect if the button is not disabled but is busy, to have a hover state - darker/lighter color, but still appearing busy. Then if the busy button is also disabled, no hover style at all.

Reply:
Fair point. I'm not quite grasping the examples, but I suppose allowing consumers to employ enabled & busy buttons is more flexible.

For the example in the OP I guess the solution would be to make the Button `busy` and `disabled`.

The styling of primary busy buttons might use a tweak, but that's one to look at separately.